### PR TITLE
[readme] Add a section to the main readme describing how to create a …

### PR DESCRIPTION
**Pull request by gottesmm**
…Swift Toolchain like we do for the swift snapshots.

### CODE REVIEW

**Comment by gottesmm** [comment]
@swift-ci smoke test

**Comment by gottesmm** [comment]
@swift-ci smoke test

**Comment by gottesmm** [comment]
@swift-ci smoke test

**Comment by gottesmm** [comment]
@swift-ci smoke test

**Comment by gottesmm** [comment]
@shahmishal ping!

File: README.md [hunk 1]
```diff
@@ -207,6 +207,40 @@ common debug flow would involve:
...
+- ``--distcc``: Use distcc to speed up the build by distributing the c++ part of the swift build.
+
+More options may be added over time. Please pass ``--help`` to
+``build-toolchain`` to see the full set of options.
+
+#### Installing into Xcode
+
+On macOS if one wants to install such a toolchain into Xcode:
+
+1. Untar and copy the toolchain to `/Library/Developer/Toolchains/`.
```
Comment (shahmishal) [inline README.md]: Users can untar the toolchain in to two location. 

1. `~/Library/Developer/Toolchains/`
2. `/Library/Developer/Toolchains/`

We should also povide command to untar. 

for example: 

`tar -zxf swift-osx.tar.gz --directory ~/`

File: README.md [hunk 2]
```diff
@@ -207,6 +207,40 @@ common debug flow would involve:
...
+
+2. Specify the local toolchain for Xcode's use via `Xcode->Toolchains`, like the
+following figure illustrates.
+
+![Specify Toolchain](docs/refactoring/Toolchain.png)
+
```
Comment (shahmishal) [inline README.md]: We should also provide information about Linux toolchain.

**Review by shahmishal — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (gottesmm) [reply README.md]: I am not trying to fix everything here. Just get something simple. Can you instead file an SR?

**Comment by gottesmm** [comment]
Just did an update.

**Review by shahmishal — APPROVED** [review]

**Pull request merged**